.circleci/config.yml

```diff
@@ -89,13 +90,66 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: v1-ruby-pupil-hpa-deps-{{ checksum "test/pupil-hpa/Gemfile.lock" }}
+          key: v2-ruby-pupil-hpa-deps-{{ checksum "test/pupil-hpa/Gemfile.lock" }}
       - run: cd test/pupil-hpa && bundle install --jobs=4 --retry=3 --path vendor/bundle
       - run: cd test/pupil-hpa && bundle exec rake -T
       - save_cache:
-          key: v1-ruby-pupil-hpa-deps-{{ checksum "test/pupil-hpa/Gemfile.lock" }}
+          key: v2-ruby-pupil-hpa-deps-{{ checksum "test/pupil-hpa/Gemfile.lock" }}
           paths:
-            - vendor/bundle
+            - test/pupil-hpa/vendor/bundle
+
+  docker_build_admin:
+    working_directory: ~/repo/
+    docker:
+      - image: cimg/base:stable
+    environment:
+      TZ: Europe/London
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 19.03.13
+      - run: cd admin && BUILDKIT=1 docker build .
+      - run: docker images
+
+  docker_build_pupil_spa:
+    working_directory: ~/repo/
+    docker:
+      - image: cimg/base:stable
+    environment:
+      TZ: Europe/London
+    steps:
+      - checkout
+      - setup_remote_docker:
+            version: 19.03.13
+      - run: cd pupil-spa && BUILDKIT=1 docker build .
+      - run: docker images
+
+  docker_build_pupil_api:
+    working_directory: ~/repo/
+    docker:
+      - image: cimg/base:stable
+    environment:
+      TZ: Europe/London
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 19.03.13
+      - run: cd pupil-api && BUILDKIT=1 docker build .
```

Comment:
wasn't aware of the new build engine, nice 💯

Reply:
It makes it a little faster, but our account with circle does not allow docker layer caching which would make a considerable difference.